bindings/core/src/method/client.rs

```diff
@@ -122,9 +123,13 @@ pub enum ClientMethod {
         query_params: Vec<String>,
         request_object: Option<String>,
     },
-    /// Build a block containing the specified payload and post it to the network.
-    PostBlockPayload {
-        /// The payload to send
+    BuildBasicBlock {
+        /// The issuer's ID.
+        issuer_id: IssuerId,
+        /// Optional strong parents for the block.
+        #[serde(default)]
+        strong_parents: Option<basic::StrongParents>,
+        /// The block payload.
         payload: PayloadDto,
```

Comment:
Do we want this optional as in the Rust function?

Reply:
Yes, actually